Azure/azure-cli
# {AppService} `az webapp`: Rename container related parameters and add parameters `--container-image-name` and `--container-registry-url` into commands `az webapp create` and `az webapp deployment slot create` to deprecate `--deployment-container-image-name`

**Related command**
`az webapp create`, `az webapp deployment slot create`, `az webapp config container set`

**Description**
Remove docker prefix and rename container webapp related parameters
`--docker-registry-server-url`
`--docker-custom-image-name`
`--docker-registry-server-user`
`--docker-registry-server-password`
to
`--container-registry-url`
`--container-image-name`
`--container-registry-user`
`--container-registry-password`
and adding parameters `--container-image-name` and `container-registry-url` into `az webapp create` and `az webapp deployment slot create` to deprecate parameter `--deployment-container-image-name`. However, we will keep `--deployment-container-image-name` and allow customer to still use it for now.

**Testing Guide**
1. Ran commands `az webapp create`, `az webapp deployment slot create`, `az webapp config container set` and check if those changes break them
2. Ran appservice tests locally

**History Notes**


[AppService] `az webapp config container set`: Remove docker prefix and rename container related parameters
[AppService] `az webapp create` and `az webapp deployment slot create`: Rename `--docker-registry-server-user` and `--docker-registry-server-password` to `--container-registry-user` and `--container-registry-password` and add parameters `--container-image-name` and `container-registry-url` to deprecate `--deployment-container-image-name`
[AppService] `az webapp create` and `az logicapp create`: Update help message

---

This checklist is used to make sure that common guidelines for a pull request are followed.
